pkg/cosign/remote.go

```diff
@@ -34,11 +47,15 @@ func Upload(signature, payload []byte, dstTag name.Reference) error {
 		}
 	}
 
+	annotations := map[string]string{
+		sigkey: base64.StdEncoding.EncodeToString(signature),
```

Comment:
maybe check for an err here?

Reply:
Where? you can't error encoding to base64.

Reply:
ack, understood (still getting used to go idioms)